kqueen/server.py

```diff
@@ -14,7 +14,22 @@
 
 import logging
 
+#Logging Configuration
+config = current_config(config_file=None)
+config.setup_policies()
+
+log_path = config.get('LOG_PATH')
+log_file = config.get('LOG_FILE')
+
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
```

Comment:
please make format configurable.

Reply:
imo there are 2 ways:
1) make config.ini file for pretty configurations
2) provide additional variables ink queen config and link it here
which one do you prefer?